### PR TITLE
[ADP-2504] program bors to use new GitHub actions

### PR DESCRIPTION
**Pull request by paolino**
- [x] remove replicated-in-buildkite GHA check `style.yml` 
- [x] update bors.toml to use the new buildkite pipelines
- [x] fix buildkite pipeline to act differently on bors branches
### Issue Number

ADP-2504

### CODE REVIEW

**Comment by paolino** [comment]
bors try

**Review by HeinrichApfelmus — APPROVED** [review]
I agree!

File: bors.toml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 status = [
-  "buildkite/cardano-wallet",
-  "ci/hydra-build:required",
+  "buildkite/cardano-wallet"
+  , "buildkite/cardano-wallet-checks-compiled-commit"
+  , "buildkite/cardano-wallet-checks-compiled-merge"
```
Comment (Anviking) [inline bors.toml]: 👍 Was about to comment we should also have these.

The "only run integration tests when pressing the button" might be inconvenient though 🤔 Now that we have bors, I suspect we should configure them to run automatically on the `bors/staging` or `bors/trying` branches

**Comment by paolino** [comment]
bors try

File: bors.toml (hunk 1 above)
Comment (paolino) [reply bors.toml]: Yes fixing that is part of this PR, the review is only functional to the work I will reiterate the request before merging, thanks

**Comment by paolino** [comment]
bors try-

**Comment by paolino** [comment]
bors try

**Comment by paolino** [comment]
bors ping

**Comment by paolino** [comment]
bors try

**Comment by paolino** [comment]
bors try

**Comment by paolino** [comment]
bors try-

**Comment by Anviking** [comment]
bors r-

^ Shouldn't work but testing anyway

**Comment by paolino** [comment]
bors try

**Comment by paolino** [comment]
bors ping

**Comment by paolino** [comment]
bors try

**Comment by paolino** [comment]
bors try

**Comment by paolino** [comment]
bors try-

**Comment by paolino** [comment]
bors try

**Comment by paolino** [comment]
bors try

**Comment by paolino** [comment]
bors ping

**Comment by paolino** [comment]
bors try

**Comment by paolino** [comment]
bors try-

**Comment by paolino** [comment]
bors ping

**Comment by paolino** [comment]
bors try

**Comment by paolino** [comment]
bors ping

**Comment by paolino** [comment]
bors try

**Comment by paolino** [comment]
bors retry

**Pull request merged**